diff --git a/imgui.cpp b/imgui.cpp
index 3c17d8ea..f0ee7f8b 100644
--- a/imgui.cpp
+++ b/imgui.cpp
@@ -18206,7 +18206,7 @@ static void ImGui::DockNodeUpdate(ImGuiDockNode* node)
     // We need to draw a background at the root level if requested by ImGuiDockNodeFlags_PassthruCentralNode, but we will only know the correct pos/size
     // _after_ processing the resizing splitters. So we are using the DrawList channel splitting facility to submit drawing primitives out of order!
     const bool render_dockspace_bg = node->IsRootNode() && host_window && (node_flags & ImGuiDockNodeFlags_PassthruCentralNode) != 0;
-    if (render_dockspace_bg && node->IsVisible)
+    if (false && render_dockspace_bg && node->IsVisible)
     {
         host_window->DrawList->ChannelsSetCurrent(DOCKING_HOST_DRAW_CHANNEL_BG);
         if (central_node_hole)
