diff --git a/rlImGui.cpp b/rlImGui.cpp
index 4a8cb3d..06e5db1 100644
--- a/rlImGui.cpp
+++ b/rlImGui.cpp
@@ -250,8 +250,10 @@ void SetupFontAwesome(void)
     ImGuiIO& io = ImGui::GetIO();
 
     float size = FONT_AWESOME_ICON_SIZE;
+    #if !defined(__APPLE__)
     if (!IsWindowState(FLAG_WINDOW_HIGHDPI))
         size *= GetWindowScaleDPI().y;
+    #endif
 
     icons_config.RasterizerMultiply = GetWindowScaleDPI().y;
 
@@ -429,10 +431,10 @@ void rlImGuiBeginInitImGui(void)
 	static constexpr int DefaultFonSize = 13;
 
     defaultConfig.SizePixels = DefaultFonSize;
-
+    #if !defined(__APPLE__)
 	if (!IsWindowState(FLAG_WINDOW_HIGHDPI))
         defaultConfig.SizePixels = ceilf(defaultConfig.SizePixels * GetWindowScaleDPI().y);
-
+    #endif
     defaultConfig.RasterizerMultiply = GetWindowScaleDPI().y;
 
     defaultConfig.PixelSnapH = true;
